src/components/RecipeEdit.vue

```diff
@@ -190,12 +190,12 @@ export default {
                 // Always set the active page last!
                 this.$store.dispatch('setPage', { page: 'edit' })
             } else {
-                this.recipe = this.recipeInit
                 this.prepTime = [0, 0]
                 this.cookTime = [0, 0]
                 this.totalTime = [0, 0]
                 this.$store.dispatch('setPage', { page: 'create' })
             }
+            this.recipeInit = this.recipe
```

Comment:
I am a bit surprised that the assignment is reversed (right/left swapped). I have not tested or debugged or whatever found all possible routes through the code but it looks suspicious to me (no offence, just an honest remark from a non-expert).

Reply:
No, I totally agree. I am confused that it ever should have worked 😅  As far as I see it, line 193 (`this.recipe = this.recipeInit`) is assigning null to the current recipe variable unless mounted() is called first. However, in this case line 193 is superfluous anyway. 

The new line 198 also shouldn’t be required, too, that's correct. The initial recipe is not used at all anyway atm and the assignment in mounted() is faulty anyway since it is only creating a reference and not a clone, but that is dealt with in #359

Reply:
A quick research shows that `mounted()` is called once Vue has initialized itself. So `recipeInit` should point to an empty recipe at the beginning.

After that statement I do not get the main point in this PR, sorry. (It might well be too late for me here...)

Reply:
What I have seen was that with `this.recipe = this.recipeInit`, the recipe variable was set to null as indicated by the screenshot below. Probably because the initial values (set in `data()`) are overwritten by the `recipeInit` variable (which is set to null in `data()`). This would be the case if `setup()` was called (e.g., from `beforeRouteEnter()`) before `mounted()` is called. Although I thought that `next()` would be called after `mounted()`.

I'll get back to this after some testing.

![Screenshot 2020-11-10 at 16 02 17](https://user-images.githubusercontent.com/7623143/98691240-1f186800-236e-11eb-9454-bb2302a1e40d.png)

Reply:
Okay, my test showed that `mounted()` was called only after setup (from `beforeRouteEnter()`). Therefore `null` was assigned to the `this.recipe` resulting in the errors.